spec/features/consumer/shopping/embedded_groups_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+feature "Using embedded shopfront functionality", js: true do
+
+  Capybara.server_port = 9999
```

Comment:
Can you check if this is already configured in `spec/spec_helper.rb`? To me, it's too unrelated to this test file to have it here and it looks like a global `Capybara` config.